full/src/main/java/apoc/sequence/Sequence.java

```diff
@@ -0,0 +1,131 @@
+package apoc.sequence;
+
+import apoc.ApocConfig;
+import apoc.SystemLabels;
+import apoc.SystemPropertyKeys;
+import apoc.util.Util;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.internal.helpers.collection.Pair;
+import org.neo4j.procedure.Context;
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.neo4j.procedure.Mode.WRITE;
+
+public class Sequence {
+
+    @Context
+    public ApocConfig apocConfig;
+
+    public static class SequenceResult {
+        public final String name;
+        public final Long value;
+
+        public SequenceResult(String name, Long value) {
+            this.name = name;
+            this.value = value;
+        }
+    }
+
+    @Procedure(mode = WRITE)
+    @Description("CALL apoc.sequence.create(name, initialValue [default=0]) - create a sequence, save it in a custom node in the system db and return it")
+    public Stream<SequenceResult> create(@Name("name") String name, @Name(value = "initialValue", defaultValue = "0") Long initialValue) {
```

Comment:
@vga91 per favore considera anche questo
```suggestion
    public Stream<SequenceResult> create(@Name("name") String name, @Name(value = "conf", defaultValue = "{}") Map<String, Object> config) {
```

con queste chiavi:
* `initialValue`: default `0`
* `createConstraint`: default `true` => sotto banco crei la constraint associata con nome `Sequence_<name>`

poi nella drop metti una conf con `dropConstraint` (default `true`)